Formula/cairo.rb

```diff
@@ -47,6 +54,7 @@ def install
       args << "--enable-xcb=no" << "--enable-xlib=no" << "--enable-xlib-xrender=no"
     end
 
+    system "./autogen.sh", *args if build.head?
```

Comment:
Indeed, after inspecting `autogen.sh`, I'm pretty sure this should be something like:

```ruby
if build.head?
  system "./autogen.sh", *args
else
  system "./configure", *args
end
```

Otherwise we're effectively running `./configure` twice for HEAD builds.